test/test-kitchen.ts

```diff
@@ -95,10 +95,15 @@ test.serial('use as a non-locally installed module', async t => {
   const GTS = `${stagingPath}/kitchen/node_modules/.bin/gts`;
   const tmpDir = tmp.dirSync({keep, unsafeCleanup: true});
   await ncpp('test/fixtures', `${tmpDir.name}/`);
+
   const opts = {cwd: `${tmpDir.name}/kitchen`};
-  // It's important to use `-n` here because we don't want to overwrite
-  // the version of gts installed, as it will trigger the npm install.
-  await simpleExecp(`${GTS} init -n`, opts);
+  await simpleExecp(`${GTS} init -y`, opts);
+  // The `extends` field must use the local gts path.
+  const tsconfigJson =
```

Comment:
nit: Could you use `pify` to promisify `readFile` so that you can `await` the `readFile`?  Also, should you be using `path.join` here and in the `t.deepEqual` line?

Reply:
This test file uses lots of `...Sync` functions. I agree that using async functions is better but since this is a test file, I think we'd better make the code more consistent. WDYT?

Reply:
I agree its not that big of a deal since its a test file, especially if there are a lot of places where `*Sync` functions are used.  If we wanted to make them async, it would probably be best as a follow-up PR.